utilities/trendQuery.js

```diff
@@ -1,15 +1,16 @@
 const googleTrends = require('google-trends-api');
-const backDateByMonth = require('./backDate');
+const backDateByMonths = require('./backDate');
+const sanitizeTrend = require('./sanitizeTrend');
 
 const trendQuery = (keyword, callback) => {
   const options = {
     keyword: keyword,
-    startTime: backDateByMonth(15),
+    startTime: backDateByMonths(15),
   };
 
   googleTrends.interestOverTime(options)
   .then((results) => {
-    callback(results);
+    callback(sanitizeTrend(JSON.parse(results)));
```

Comment:
@aseiden I thought about that too. I guess I was wondering if we should assume that the `googleTrends.interestOvertime` returns an object but it is already designed the way it is and it gives a string so I am actually 100% for moving the parse to the `sanitizeTrend` so I'll do that in re-submit 👯‍♂️